doc/API-Mastodon.md

```diff
@@ -106,11 +194,22 @@ These endpoints use the [Mastodon API entities](https://docs.joinmastodon.org/en
 - [`DELETE /api/v1/scheduled_statuses/:id`](https://docs.joinmastodon.org/methods/statuses/scheduled_statuses/)
 - [`GET /api/v1/scheduled_statuses/:id`](https://docs.joinmastodon.org/methods/statuses/scheduled_statuses/)
 - [`GET /api/v1/search`](https://docs.joinmastodon.org/methods/search/)
+- [`PUT /api/v1/statuses`](https://docs.joinmastodon.org/methods/statuses/#edit)
+    - Does not support `polls` argument as Friendica does not have polls
+	- Additional fields `friendica` for Friendica specific parameters:
+		- `title`: Explicitly sets the title for a post status, ignored if used on a comment status. For post statuses the legacy behavior is to use any "spoiler text" as the title if it is provided. If both the title and spoiler text are provided for a post status then they will each be used for their respective roles. If no title is provided then the legacy behavior will persist. If you want to create a post with no title but spoiler text then explicitly set the title but set it to an empty string `""`.
```

Comment:
Indentation is different than the previous lines.

Reply:
The indentation for the friendica line is wrong. The indentation for the title line is supposed to show it is a sub field of friendica.

Reply:
Now that I'm on desktop, I see there is a mix of spaces and tabs for indentation.

Reply:
I thought my PHPStorm was setup to convert tabs to spaces, I guess not.

Reply:
...is the indentation for sub fields appropriate or is there another way we prefer to do this?

Reply:
I believe it’s fine.